### ISSUE TITLE
REDMINE_PLUGINS_MIGRATE

### ISSUE THREAD

**Issue by kebyn** [docker-library/redmine#75]
`REDMINE_PLUGINS_MIGRATE`This variable is invalid in version 3.3.3.
This `docker-entrypoint.sh` part was deleted:
```
if [ "$1" != 'rake' -a -n "$REDMINE_PLUGINS_MIGRATE" ]; then 
 gosu redmine rake redmine:plugins:migrate 
fi 
```
please tell me ,why?
thanks!

**Comment by yosifkit** [comment]
According to the git log it was never added.  It was added to the master entrypoint script in the root directory (https://github.com/docker-library/redmine/pull/10, merged on Apr 25), but somehow when our bot ran `update.sh` it never committed the change to the entrypoint scripts.  I checked on the bot so that this won't happen in the future and it seems it [only commits `Dockerfile` changes](https://github.com/docker-library/oi-janky-groovy/blob/001b245a293dbc8416bff7951a9053bb8fac62c6/update.sh/target-pipeline.groovy#L140). :cry:  I'll go work on a fix.
